common/p_spec.cpp

```diff
@@ -2146,7 +2326,7 @@ void DScroller::RunThink ()
 	if (m_Control != -1)
 	{	// compute scroll amounts based on a sector's height changes
 		sector_t *sector = &sectors[m_Control];
-		fixed_t height = P_FloorHeight(sector->soundorg[0], sector->soundorg[1], sector);
+		fixed_t height = sector->ceilingheight;
```

Comment:
This seems incorrect.  Why are we getting rid of `P_FloorHeight` here?  Do we need the raw value here for some reason?

Reply:
These (scrollers that use control sectors) never worked correctly in Odamex. The Boom scrollers that use control sectors depend on _ceiling_ height changes, and the initial height is recorded during map load and compared then on. Consider the following, line special 215 (Scroll Floor Accelerates When Sector Changes Height). This is its behavior on Odamex 0.9.5:
https://youtu.be/Qzf6i4oWI2M
Behavior on the commit that I haven't pushed yet, but didn't modify this code:
https://youtu.be/GnAreXbAfXM
And behavior for DSDA-Doom:
https://youtu.be/qqnxemrrb9A
It looks odd but we had the spec wrong from the beginning, and I believe mine is correct to the best of my knowledge.